### ISSUE TITLE
Speedup 10.11 boot by disabling some LaunchDaemons

### ISSUE THREAD

**Issue by macmule** [macmule/AutoCasperNBI#78]
```
rm -r  "${TMP_MOUNT_PATH}"/System/Library/LaunchDaemons/com.apple.locationd.plist
rm -r  "${TMP_MOUNT_PATH}"/System/Library/LaunchDaemons/com.apple.lsd.plist
rm -r  "${TMP_MOUNT_PATH}"/System/Library/LaunchDaemons/com.apple.ocspd.plist
rm -r  "${TMP_MOUNT_PATH}"/System/Library/LaunchDaemons/com.apple.tccd.system.plist
rm -r  "${TMP_MOUNT_PATH}"/System/Library/LaunchDaemons/org.ntp.sntp.plist
```

from DS 1.7.2

also https://github.com/NBICreator/NBICreator/blob/master/NBICreator/Workflow%20Modify/NBCWorkflowResourcesModify.m#L1124-L1141

**Comment by hunty1** [comment]
Don't remove `com.apple.lsd.plist` 
that will result in the following: 

```root# open /Applications/Utilities/Terminal.app/
The application cannot be opened because its executable is missing.```

Basically unable to open any applications from the NBI using terminal commands.

Probably an edge case, but for me it screwed me up as i run some scripts first which open other applications or allow the user to click buttons which opens other applications. and having this lsd daemon removed prevents it from working :)

**Comment by macmule** [comment]
Hey fella,

Yea. Trying it did not give desired results.

There is a JAMFNation thread with folks having calls with Apple about the speed. Seems like an 10.11 issue & they see the same with SIU.

Awaiting the resolution in that thread before doing anything with ACNBI. 

Regards,

Ben.

**Comment by macmule** [comment]
Resolved via 10.11.6, so closing this off.

[10.11.6 Release Notes](https://support.apple.com/en-us/HT206770)
